sound/soc/intel/boards/sof_sdw.c

```diff
@@ -456,6 +456,15 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		},
 		.driver_data = (void *)(RT711_JD2_100K),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Google"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "Rex"),
+		},
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
```

Comment:
SOF_SDW_TGL_HDMI is not needed. We decide to export 3 HDMI only.

Reply:
Removed SOF_SDW_TGL_HDMI and updated the commit